data/faqs.yml

```diff
@@ -29,7 +29,7 @@ items:
       There will always be a role for gateways, but we anticipate that role will shrink as direct
       IPFS support becomes more prevalent.
   - name: |
-      How does the gateway <a href=https://docs.ipfs.tech/reference/http/gateway/#trustless-verifiable-retrieval" target="_blank">ensure</a> whether the data they return matches the Content ID (CID) in the request? Can I access the gateways in a trustless way?
```

Comment:
Links don't seem to work in titles here

Reply:
Makes sense, it's a heading, so not surprised it isn't allowing links in there.